insights/specs/default.py

```diff
@@ -452,6 +452,7 @@ def semid(broker):
     ls_var_lib_mongodb = simple_command("/bin/ls -la /var/lib/mongodb")
     ls_R_var_lib_nova_instances = simple_command("/bin/ls -laR /var/lib/nova/instances")
     ls_var_lib_nova_instances = simple_command("/bin/ls -laRZ /var/lib/nova/instances")
+    ls_var_opt_mssql = simple_command("/bin/ls -laR /var/opt/mssql")
```

Comment:
@psachin - is it necessary to `recursively` list all the items in this dir? I'm afraid the size with `-R` would be too large.

Does the `ls -la /var/opt/mssql` work for you?